### PR TITLE
Refs 2471 Update Resource Image Details Screeen

### PR DESCRIPTION
**Pull request by ducvu212**
# Sumary
-
# Redmine
- https://edu-redmine.framgia.vn/issues/2471?issue_count=63&issue_position=1&next_issue_id=2361
# Buglist
-

### CODE REVIEW

**Comment by ducvu212** [comment]
ready

**Comment by ducvu212** [comment]
ready

**Comment by ducvu212** [comment]
ready

**Comment by ducvu212** [comment]
ready

**Comment by ducvu212** [comment]
ready

File: app/src/main/res/layout/item_random_image.xml [hunk 1]
```diff
@@ -5,14 +5,19 @@
 
     <data>
 
+        <variable
+            name="listener"
+            type="com.example.ducvu212.demomvvm.screen.home.HandleItemClick"/>
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_random_image.xml]: (tat) com.example.ducvu212.demomvvm

**Comment by tuanlvt-1117** [comment]
commented

**Comment by ducvu212** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Comment by tuanlvt-1117** [comment]
merged

**Pull request merged**